docs/source/rdump.rst

```diff
@@ -0,0 +1,115 @@
+
+    $ target-query host.img -f users | rdump -F name,home
+
+Using the ``-X`` option we can exclude fields:
+
+    $ target-query host.img -f users | rdump -X hostname,domain,sid
+
+
+Filtering
+~~~~~~~~~
+
+Filtering records can be done through the ``-s`` option.
+The selection option must be Python-expression, where the record is represented by the symbol ``r``.
+So to eliminate all records that have no domain value:
+
+.. code-block:: console
+
+    $ target-query host.img -f users | rdump -F name,home -s "r.domain is not None"
+
+
+Derivations
+~~~~~~~~~~~
+
+It is possible to create new fields derived from other fields. This can be done by using
+an expression and the ``-E`` option. The following example lists all dll files from
+a host and adds the filename to a separate field:
+
+.. code-block:: console
+
+    $ target-query host.img -f walkfs | rdump -s "r.path.suffix=='.dll'" -F path,file,size -E "file=path.name"
+    
+    <filesystem/entry path='\sysvol\Program Files\Ethereal\libgdk_pixbuf-2.0-0.dll' size=0.11 MB file='libgdk_pixbuf-2.0-0.dll'>
+    <filesystem/entry path='\sysvol\Program Files\Ethereal\libgmodule-2.0-0.dll' size=26.6 KB file='libgmodule-2.0-0.dll'>
+    <filesystem/entry path='\sysvol\Program Files\Ethereal\libgobject-2.0-0.dll' size=0.25 MB file='libgobject-2.0-0.dll'>
+    ....and many more....
+
+
+Limiting
+~~~~~~~~
+
+The example above yields quite a lot records. Limiting the number of records can be done through
+the ``--count`` and ``--skip`` options:
+
+.. code-block:: console
+
+    $ target-query host.img -f walkfs | rdump -s "r.path.suffix=='.dll'" -F path,file,size -E "file=path.name" --skip=1 --count=3
+
+    <filesystem/entry path='\sysvol\My Documents\COMMANDS\cygwinb19.dll' size=0.38 MB file='cygwinb19.dll'>
+    <filesystem/entry path='\sysvol\My Documents\ENUMERATION\NT\Cerberus\dnsscan.dll' size=40.0 KB file='dnsscan.dll'>
+    <filesystem/entry path='\sysvol\My Documents\ENUMERATION\NT\Cerberus\fingerscan.dll' size=40.0 KB file='fingerscan.dll'>
+
+Formatting
+~~~~~~~~~~
+
+A Python-style formatting rule can be specified using the ``-f`` option. The following example applies the format
+``{file} ({size})`` to the records:
+
+.. code-block:: console
+
+    $ target-query host.img -f walkfs | rdump -s "r.path.suffix=='.dll'" -F path,file,size -E "file=path.name" --skip=1 --count=3 -f "{file} ({size})"
+    
+    cygwinb19.dll (0.38 MB)
+    dnsscan.dll (40.0 KB)
+    fingerscan.dll (40.0 KB)
+
+Instead of having to design your own format you can also choose one of these
+
+* JSON (``-j`` or ``--mode=json``)
+* CSV (``-C`` or ``--mode=csv``)
+* Line (``-L`` or ``--mode=line``)
+
+It is also possible let an adapter take care of the formatting. For instance, if you wish to have your
+records in an archive format with a year-month-day folder structure, you can employ the ``-w`` option and
+choose the archive adapter: ``archive://outputdir``. For complete list of adapters use ``-a``.
```

Comment:
Great that the -w parameter is discussed, however I would propose to highlight another adapter that an analyst is more likely to use. Such as Splunk or Elastic.

Reply:
But also more complicated, as it requires additional knowledge about these systems. While this adapter is less frequently used, it is more suitable for demonstration purposes. The rest can be covered in the advanced sections.